src/qibolab/channels.py

```diff
@@ -105,12 +105,12 @@ def power_range(self, value):
         self.port.power_range = value
 
     @property
-    def filters(self):
-        return self.port.filters
+    def filter(self):
```

Comment:
I know, I just wanted to use exactly the same name QM is using, since it's the only driver supporting this and it's easier to keep the same name. I does not cause any issue so far (since you always access as `.filter` and never as `filter`).